Dockerfile

```diff
@@ -60,6 +63,8 @@ ENV \
 COPY --from=builder --chown=node:node /directus/ecosystem.config.cjs .
 COPY --from=builder --chown=node:node /directus/dist .
```

Comment:
Technical we don't need `--chown=node:node` here anymore. But it also does not hurt ;D